src/_pytest/config/__init__.py

```diff
@@ -137,6 +137,24 @@ def main(args=None, plugins=None) -> Union[int, ExitCode]:
         return ExitCode.USAGE_ERROR
 
 
+def console_main() -> int:
```

Comment:
Can we test this somehow? Perhaps with `testdir.runpytest_subprocess`?

Reply:
Let me try...